Cannot find module 'tslib'

### Describe the bug

I have followed your default example in the readme, however, as soon as I try to run npx tailwindcss I get the following error

```
"C:\Program Files\nodejs\npm.cmd" run site-css

> site-css
> npx tailwindcss -i tailwind/styles.css -o wwwroot/css/styles.css --minify --watch


Rebuilding...
Error: Cannot find module 'tslib'
Require stack:
- C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\node_modules\tailwindcss-themer\lib\index.js
- C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\tailwind.config.js
- C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\node_modules\tailwindcss\lib\cli\build\plugin.js
- C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\node_modules\tailwindcss\lib\cli\build\index.js
- C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\node_modules\tailwindcss\lib\cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\Users\lee\OneDrive\Documents\Projects\Blog\Blog.Web\node_modules\tailwindcss-themer\lib\index.js:2:17)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\lee\\OneDrive\\Documents\\Projects\\Blog\\Blog.Web\\node_modules\\tailwindcss-themer\\lib\\index.js',
    'C:\\Users\\lee\\OneDrive\\Documents\\Projects\\Blog\\Blog.Web\\tailwind.config.js',
    'C:\\Users\\lee\\OneDrive\\Documents\\Projects\\Blog\\Blog.Web\\node_modules\\tailwindcss\\lib\\cli\\build\\plugin.js',
    'C:\\Users\\lee\\OneDrive\\Documents\\Projects\\Blog\\Blog.Web\\node_modules\\tailwindcss\\lib\\cli\\build\\index.js',
    'C:\\Users\\lee\\OneDrive\\Documents\\Projects\\Blog\\Blog.Web\\node_modules\\tailwindcss\\lib\\cli.js'
  ]
}

```


### Your minimal, reproducible example

privaterepo

### Steps to reproduce

Just follow your example on the read me

### Expected behavior

I expected it to build the tailwind CSS file with the new theme classes

### How often does this bug happen?

None

### Screenshots or Videos

_No response_

### Platform

Windows 11

### tailwindcss-themer version

Latest

### Additional context

_No response_

--- a/package.json
+++ b/package.json
@@ -78,7 +78,6 @@
     "stylelint-prettier": "^2.0.0",
     "tailwindcss": "^3.2.7",
     "ts-jest": "^29.0.5",
-    "tslib": "^2.5.0",
     "typescript": "^4.9.5"
   },
   "peerDependencies": {

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -52,7 +52,7 @@
     "outDir": "lib" /* Specify an output folder for all emitted files. */,
     // "removeComments": true,                           /* Disable emitting comments. */
     // "noEmit": true,                                   /* Disable emitting files from a compilation. */
-    "importHelpers": true /* Allow importing helper functions from tslib once per project, instead of including them per-file. */,
+    // "importHelpers": false                            /* Allow importing helper functions from tslib once per project, instead of including them per-file. */,
     // "importsNotUsedAsValues": "remove",               /* Specify emit/checking behavior for imports that are only used for types */
     // "downlevelIteration": true,                       /* Emit more compliant, but verbose and less performant JavaScript for iteration. */
     // "sourceRoot": "",                                 /* Specify the root path for debuggers to find the reference source code. */

